src/plugins/image-tiles/display/image-tiles-display.js

```diff
@@ -26,7 +26,7 @@ function createTileUrl(tile) {
     case LINK_TYPE.external:
       return link.url;
     case LINK_TYPE.article:
```

Comment:
Will `LINK_TYPE.article` still exist actually? I think we probably need a migration script to adjust this plus the URLs, right?

Reply:
Yes, I discussed that with Andreas a couple of days ago and we will fix this component in another ticket. The ticket is already made.
https://educandu.atlassian.net/browse/EDU-150